src/google_map_markers.js

```diff
@@ -229,6 +229,7 @@ export default class GoogleMapMarkers extends Component {
     this.dimesionsCache_ = {};
 
     const markers = React.Children.map(this.state.children, (child, childIndex) => {
+      if (!child) return null;
```

Comment:
I don't like `null`s ;-) the same result we will get with `undefined`. 
So the question was about does `null` here has any advantage or hidden sense over `undefined`. 

Having ES6 using `null` in any chain like env prevents me to use arguments defaults in the future, so I prefer not to use `null` at all. 

So please change null on undefined ;-)